content-src/components/DiscoveryStreamBase/DiscoveryStreamBase.jsx

```diff
@@ -157,6 +157,8 @@ export class _DiscoveryStreamBase extends React.PureComponent {
         return (
           <ImpressionStats rows={rows} dispatch={this.props.dispatch} source={component.type}>
             <Hero
+              subComponentType={embedWidth >= 9 ? `cards` : `list`}
```

Comment:
personally I would pass in row width and add this logic to `Hero` if we intend to keep it consistent across usages

Reply:
Yeah, I can see it both ways I guess. I'm trying to keep the components fairly dumb and light on logic.

Reply:
This is also the only way I can foresee Hero being embedded.